version.json

```diff
@@ -1,7 +1,6 @@
 {
-    "build": null,
-    "commit": "HEAD",
     "source": "https://github.com/mozilla/ichnaea",
-    "tag": "master",
-    "version": "dev"
+    "version": "unknown",
+    "commit": "HEAD",
+    "build": "unknown"
```

Comment:
What is this new key order?

Reply:
It matches what's in the spec. Figured it's easier to eyeball:

https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md